src/main/java/duke/Event.java

```diff
@@ -0,0 +1,27 @@
+package duke;
+
+/**
+ * duke.Event class to create Events for the duke list
+ */
+
+public class Event extends Task {
+    protected String time;
+
+    public Event(String description, String time) {
+        super(description);
+        this.time = time;
+    }
+
+    /**
+     * To get the time of the event stored in the list
```

Comment:
Javadoc headers should start with words in the form `Returns ...`, `Sends ...`, `Adds ...` (not `Return` or `Returning` etc.)
This applies to your other javadoc headers too :)